pkg/deploy/api/v1/register.go

```diff
@@ -9,9 +9,11 @@ func init() {
 		&DeploymentConfig{},
 		&DeploymentConfigList{},
 		&DeploymentConfigRollback{},
+		&Scale{},
 	)
 }
 
 func (*DeploymentConfig) IsAnAPIObject()         {}
 func (*DeploymentConfigList) IsAnAPIObject()     {}
 func (*DeploymentConfigRollback) IsAnAPIObject() {}
+func (*Scale) IsAnAPIObject() {}
```

Comment:
Since this is going to be a dc subresource I would got with `DeploymentConfigScale`.